source/_integrations/assist_satellite.markdown

```diff
@@ -0,0 +1,45 @@
+---
+title: Assist Satellite
+description: Remote satellites that use Assist.
+ha_category:
+  - Voice
+ha_release: '2024.10'
+ha_codeowners:
+  - '@balloob'
+  - '@synesthesiam'
+ha_domain: assist_satellite
+ha_integration_type: entity
+ha_quality_scale: internal
+---
+
+Platform for remote satellites that use [Assist](/voice_control/) to control and interact with Home Assistant. This {% term integration %} allows other integrations to represent these satellites in a consistent manner.
```

Comment:
It's not a platform. Other integrations will implement a platform under this domain.

```suggestion
This is an integration for remote satellites that use [Assist](/voice_control/) to control and interact with Home Assistant. This {% term integration %} allows other integrations to represent these satellites in a consistent manner.
```

Reply:
Maybe rewrite it to decrease some repetition of "this integration".